implementation/libs/jbb-lib-mvc/src/main/java/org/jbb/lib/mvc/repository/JbbSessionRepository.java

```diff
@@ -0,0 +1,75 @@
+package org.jbb.lib.mvc.repository;
```

Comment:
Package is a bit weird for me... maybe `org.jbb.lib.mvc.session` instead `org.jbb.lib.mvc.repository`?